modtools/pref-edit.lua

```diff
@@ -0,0 +1,326 @@
+-- Add, remove, or edit the preferences of a unit
+-- Not to be confused with "pref_adjust" and "prefchange"
+--@ module = true
+
+local help = [====[
+
+modtools/pref-edit
+==================
+Add, remove, or edit the preferences of a unit.
+Requires a modifier, a unit argument, and filters.
+
+:unit <UNIT ID>:
```

Comment:
Afaik `[]` is used for optional arguments

Reply:
I checked the style guide but couldn't find anything about using square brackets (though did find I'm not sticking to them exactly, oops :P). It's kinda confusing as to whether the unit id is an optional addition because of how the scripts handle missing/invalid targets.

Reply:
Yeah, that's probably something we could put in our documentation (I'll try to find a good place for it)

Reply:
It's more of a "unix" convention or just general commandline tool convention.